admin/users/views.py

```diff
@@ -90,21 +91,20 @@ def delete(self, request, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         context = {}
-        context.setdefault('guid', kwargs.get('object').pk)
+        context.setdefault('guid', kwargs.get('object')._id)
         return super(UserDeleteView, self).get_context_data(**context)
 
     def get_object(self, queryset=None):
         return OSFUser.load(self.kwargs.get('guid'))
 
 
-class SpamUserDeleteView(UserDeleteView, PermissionRequiredMixin):
+class SpamUserDeleteView(UserDeleteView):
```

Comment:
On second thought, not gonna include those extra permissions I don't think cause it's not really a delete, persay, but rather a disable. So should just stick with `DeleteUser`'s `osf.change_user` permission